libs/components/board-draw/src/TlDraw.tsx

```diff
@@ -361,11 +361,11 @@ const InnerTldraw = memo(function InnerTldraw({
     const isInSession = app.session !== undefined;
 
     // Hide bounds when not using the select tool, or when the only selected shape has handles
-    const hideBounds =
-        (isInSession && app.session?.constructor.name !== 'BrushSession') ||
-        !isSelecting ||
-        isHideBoundsShape ||
-        !!pageState.editingId;
+    // const hideBounds =
+    //     (isInSession && app.session?.constructor.name !== 'BrushSession') ||
+    //     !isSelecting ||
+    //     isHideBoundsShape ||
+    //     !!pageState.editingId;
```

Comment:
Should the selected bounds be displayed in all cases? For example, during brush painting

Reply:
NO, i will fix the bug